app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -107,6 +107,7 @@ export const SearchPage: React.FC = () => {
           makeFilter({
             ...filter,
             contentTypes: [],
+            pageSize: 10000,
```

Comment:
I'm not a fan of this solution, but for now it'll have to do.
Essentially we're asking Elasticsearch for the largest number of hits possible every time.  Ideally we'd only ask for a realistic number of stories and page through results.

Reply:
@Fosol this one ready for review now as well - although it is a "for now" solution as discussed